### ISSUE TITLE
False positive regarding github.repository.name

### ISSUE THREAD

**Issue by mm-chia** [rhysd/actionlint#529]
Hello,

`actionlint` reports error when encountering `${{ github.repository_name }}` but this property is perfectly valid.
I think the official Github documentation is lacking on this.

**Comment by muzimuzhi** [comment]
I did a test and it shows `github.repository_name` doesn't exist.

https://github.com/muzimuzhi/hello-github-actions/actions/runs/17893422274/job/50876875865

```
Run echo "${{ github.repository_name }}"
  echo "${{ github.repository_name }}"
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/b3eacd56-5f2a-42cc-b31d-8bc756052394.sh: line 1: ${{ github.repository_name }}: bad substitution
Error: Process completed with exit code 1.
```

**Comment by rhysd** [comment]
I guess `repository_name` did exist when this issue was created. However, as it was undocumented, I guess it was for GitHub internal use and thus removed later. Closing.
